cmd/containerd-shim-runc-v2/task/service.go

```diff
@@ -298,6 +308,8 @@ func (s *service) Start(ctx context.Context, r *taskAPI.StartRequest) (*taskAPI.
 			if err := s.ep.Add(container.ID, cg); err != nil {
 				log.G(ctx).WithError(err).Error("add cg to OOM monitor")
 			}
+		case nil:
+			log.G(ctx).Debug("nil cgroup for container", container.ID)
```

Comment:
why? 
accessing to `cg` doesn't seem to occur after this line.

Reply:
This is to make it consistent with the other cases, which log a debug line in case that there's no cgroup. I thought the logs might be a helpful addition when troubleshooting. I can also remove the logging completely, just leaving the nil checks in place, whatever you prefer.